planner/core/point_get_plan.go

```diff
@@ -1280,6 +1280,11 @@ func findInPairs(colName string, pairs []nameValuePair) int {
 }
 
 func tryUpdatePointPlan(ctx sessionctx.Context, updateStmt *ast.UpdateStmt) Plan {
+	for _, list := range updateStmt.List {
+		if _, ok := list.Expr.(*ast.SubqueryExpr); ok {
```

Comment:
better add a comment about this? when update list contains subquery, avoid point get plan

Reply:
Have added